### DISCUSSION TITLE
link to a local file on local filesystem?

### DISCUSSION THREAD

**Discussion by thankstipscom** [requarks/wiki#5802] [Help / Questions] [answered]
I have wikijs running in docker. I have the local filesystem enabled with the target and the backups. This all works fine.

I have a bunch of videos (meetings and such) that I dont want hosted publicly but the docker server has access to the folder where these videos are. I've mounted it in the docker compose file and in the wikijs image I can browse to the folder. 

Last step which I can not figure out for the life of me, is how can I just link to the file in a wiki page? I can only seem to link to a file in the virtual folders. I dont want to be uploading 2gb videos to local storage, especially when the video is literally RIGHT there accessible locally. How can I (preferably in markdown but any way is fine) linke on a wiki page to a folder on a local file system that the image and the docker server have access to? Its not hosted on a website, I wanna link to the file directly.

It seems like a simple thing, DokuWiki was able to do it... Is this just not possible with Wikijs??

thanks.

**Comment by yangboz** [comment]
and currently in yarn dev model ,it depends on PostgreSQL 5432 port .
if not 
```
2022-11-02T02:11:19.588Z [MASTER] info: Connecting to database...
2022-11-02T02:11:19.593Z [MASTER] error: Database Initialization Error: connect ECONNREFUSED 127.0.0.1:5432
2022-11-02T02:11:19.593Z [MASTER] error: connect ECONNREFUSED 127.0.0.1:5432

**Comment by NGPixel** [comment, marked as answer]
That's not possible no.
